src/application/engine/entities/BoDCharacter.java

```diff
@@ -37,6 +37,11 @@
      * @param speed
      *            Movement speed of the character.
      */
+    public BoDCharacter(int id)
```

Comment:
Our idea was to make character abstract. The reason we set them in the specializations is because it is on these objects that a specializations stats and balancing is done.